patcher/src/widgets/Controller/components/PatchButton/index.tsx

```diff
@@ -268,4 +274,4 @@ class PatchButton extends React.Component<PatchButtonProps, PatchButtonState> {
   }
 }
 
-export default PatchButton;
+export default connect()(PatchButton);
```

Comment:
What is the purpose of this connect? The component should not be wrapping in a connect call, if you just want dispatch in the component accept the function as a prop.